tests/foreman/cli/test_remoteexecution.py

```diff
@@ -629,14 +630,18 @@ def test_positive_run_concurrent_jobs(self, fixture_vmsetup, module_org):
         GlobalParameter().delete({'name': param_name})
         assert len(GlobalParameter().list({'search': param_name})) == 0
 
+    nick_params = [{'nick': 'rhel7'}, {'nick': 'rhel7_fips'}, {'nick': 'rhel8'}]
+    if not is_open('BZ:1811166'):
+        nick_params.append({'nick': 'rhel8_fips'})
```

Comment:
I somehow do not like this section. I know that this is just a backport, but I would like to ask @peterdragun to have look at this as he played around with fixture and test parametrization lately.

Reply:
I don't think we have any better option here unfortunately. Other option would be skip test with this parameter directly in test function but this would be even worse IMO.

Reply:
Thanks!